front/src/modules/activities/tasks/hooks/useTasks.ts

```diff
@@ -68,6 +40,7 @@ export function useTasks(entity?: ActivityTargetableEntity) {
         ...whereFilters,
       },
     },
+    skip: filters.length === 0,
```

Comment:
For some reason, apollo is not picking the last where filters correctly. It seems to be a race condition somewhere that I could not debug properly. I think to avoid running into this kind of issue, we should only instantiate TasksGroups when its context (here the tasks filters) is ready